packages/mobile/src/send/SendAmount.tsx

```diff
@@ -153,10 +153,7 @@ function SendAmount(props: Props) {
   const isDollarBalanceSufficient = isAmountValid && newAccountBalance.isGreaterThanOrEqualTo(0)
 
   const reviewBtnDisabled =
-    !isAmountValid ||
-    (isRequest
```

Comment:
if `isRequest` and address isn't verified we will now route to an error screen instead of keeping the button disabled